test/parallel/test-timer-close.js

```diff
@@ -1,16 +1,10 @@
 'use strict';
-require('../common');
-var assert = require('assert');
+const common = require('../common');
 
-var t = new (process.binding('timer_wrap').Timer)();
-var called = 0;
-function onclose() {
-  called++;
-}
+// Make sure handle._handle.close(callback) is idempotent by closing a timer
+// twice. The first function should be called, the second one should not.
 
-t.close(onclose);
-t.close(onclose);
+const t = new (process.binding('timer_wrap').Timer)();
```

Comment:
hmm... can't say that I'm really a fan of this style (even tho I know it was used before this change)... I prefer separating out the call to process.binding...

```js
const Timer = process.binding('timer_wrap').Timer;
const t = new Timer();
```